wrappers/python/tests/agent/test_agent_send.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import pytest
+
+from indy import signus, ledger, agent
+from tests.utils import wallet
+
+
+@pytest.mark.asyncio
+async def test_agent_send_works_for_all_data_in_wallet_present(connection):
```

Comment:
Do we have a test to check a pairwise connection, that is communication in both directions (with replies)?